tests/functional/contribute/test_events.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from ...pages.contribute.events import ContributeEventsPage
+
+
+@pytest.mark.nondestructive
+def test_events_table_is_displayed(base_url, selenium):
+    page = ContributeEventsPage(base_url, selenium).open()
+    assert page.events_table_is_displayed
+
+
+@pytest.mark.nondestructive
+def test_next_event_is_displayed(base_url, selenium):
+    page = ContributeEventsPage(base_url, selenium).open()
+    assert page.next_event_is_displayed
```

Comment:
The total at the top of the page is hard coded, not dynamic. I don't think we need to go as far as covering that each field from the DB is populated in the template. If there was an issue with the cron or elsewhere, this region would not get displayed.